vizro-ai/tests/unit/vizro-ai/dashboard/_response_models/test_df_info.py

```diff
@@ -0,0 +1,7 @@
+from vizro_ai.dashboard._response_models.df_info import _get_df_info
+
+
+def test_get_df_info(df, df_schema):
+    actual_df_schema, _ = _get_df_info(df=df)
```

Comment:
The return type is "Tuple[Dict[str, str], pd.DataFrame]"
so the return sample dataset can be tested too